src/network/network.cpp

```diff
@@ -1100,13 +1100,53 @@ void NetworkShutDown()
 	NetworkCoreShutdown();
 }
 
+/**
+ * Get the network version string used by this build.
+ * The returned string is guaranteed to be at most NETWORK_REVISON_LENGTH bytes.
+ */
+const char * GetNetworkRevisionString()
+{
+	/* This will be allocated on heap and never free'd, but only once so not a "real" leak. */
+	static char *network_revision = nullptr;
+	/* We want at least this many characters of the git hash, if the base string becomes truncatad. */
+	const uint MIN_GITHASH_LENGTH = 8;
```

Comment:
Unsure if this constant is good. It's the same as used in the findversion/determineversion scripts, but doesn't match the shorthash git itself uses, which is 7 characters for me right now.